staging/src/k8s.io/kubectl/pkg/cmd/wait/wait.go

```diff
@@ -380,7 +400,7 @@ func (w ConditionalWait) IsConditionMet(info *resource.Info, o *WaitOptions) (ru
 			resourceVersion = gottenObjList.GetResourceVersion()
 		default:
 			gottenObj = &gottenObjList.Items[0]
-			conditionMet, err := w.checkCondition(gottenObj)
+			conditionMet, err := w.checkConditions(gottenObj)
```

Comment:
```suggestion
			conditionsMet, err := w.checkConditions(gottenObj)
```

Reply:
Thanks for pointing it out . Will change this